packages/css-library/.gitignore

```diff
@@ -1,5 +0,0 @@
-dist
```

Comment:
Because it looked like some tools would benefit from having the a token file available by linking to it directly, I'm removing the ignore for `dist` and generating them here.